query/aql_compiler.go

```diff
@@ -1583,11 +1589,11 @@ func (qc *AQLQueryContext) processTimeFilter() {
 				qc.Query.TimeFilter.Column)
 			return
 		}
-		timeColumnType := qc.TableScanners[0].Schema.Schema.Columns[timeColumnID].Type
-		if timeColumnType != metaCom.Uint32 {
+		timeColumnType := qc.TableScanners[0].Schema.ValueTypeByColumn[timeColumnID]
```

Comment:
why don't we should delete `metastore/common/data_type.go`, and move everything to `memstore/common`, it only defines string constants. `memstore/common` should be safe to import from anywhere.

Reply:
let's revisit this in a different issue